idb/postgres/internal/schema/setup_postgres.sql

```diff
@@ -69,6 +69,8 @@ CREATE TABLE IF NOT EXISTS account_app (
   addr bytea,
   app bigint,
   localstate jsonb NOT NULL, -- json string "null" iff deleted from the account
+  voting_start BIGINT, -- voting start
+  voting_end BIGINT, -- voting end
```

Comment:
let's create another table: `proposals`, and put proposal params there. It will be `1-*`  (one account, possible many proposals). So whenever a new account is created for a proposal, or new proposal is created for an existing account we will update the proposals table. Moreover, we should not delete proposals, instead we can display latest, and past proposals.

Reply:
Pleased check DM.